classes/clock/clock.js

```diff
@@ -27,7 +27,7 @@ class Clock extends Time {
         if (this.interval) return;
 
         this.interval = setInterval(() => {
-            if (this.totalSeconds === MAX_CLOCK_SECONDS) this.pause();
+            if (this.tSeconds === MAX_CLOCK_SECONDS) this.pause();
             else this.addSeconds(1);
```

Comment:
I recommend to use `super.addSeconds(1)` although the **this** will work as well 
I think it's nicer to see where this method came from the inheritance..

Reply:
so should i always use super in this cases?

Reply:
changed